chromium_src/chrome/browser/renderer_context_menu/render_view_context_menu.cc

```diff
@@ -501,6 +511,27 @@ void BraveRenderViewContextMenu::ExecuteCommand(int id, int event_flags) {
       cosmetic_filters::CosmeticFiltersTabHelper::LaunchContentPicker(
           source_web_contents_);
       break;
+    case IDC_NEW_EMAIL_ALIAS:
+      if (params_.form_control_type.value() ==
+              blink::mojom::FormControlType::kInputEmail ||
+          params_.form_control_type.value() ==
+              blink::mojom::FormControlType::kInputText ||
+          params_.is_content_editable_for_autofill) {
+        if (GetProfile()
+                ->GetPrefs()
+                ->GetString(kEmailAliasesAccountEmail)
+                .empty()) {
+          brave::ShowEmailAliases(GetBrowser());
+        } else {
+          auto* browser_view =
+              BrowserView::GetBrowserViewForBrowser(GetBrowser());
+          views::View* anchor_view = browser_view->GetLocationBarView();
+          email_aliases::EmailAliasesBubbleUI::Show(GetBrowser(), anchor_view,
```

Comment:
shouldn't this have a feature flag guard?

Reply:
added